src/formats/sprite.c

```diff
@@ -308,7 +311,13 @@ int sd_sprite_vga_decode(sd_vga_image *dst, const sd_sprite *src) {
             case 1:
                 while(data > 0) {
                     uint8_t b = src->data[i];
-                    int pos = ((y * src->width) + x);
+                    unsigned int pos = ((y * src->width) + x);
+                    // if we're about to overwrite the `dst` buffer, don't.
```

Comment:
```suggestion
                    // if we're about to overflow the `dst` buffer, don't.
```

Reply:
i will apply this wording change when i know what to do for the other conversation